### PR TITLE
scroll bar fix in incanvas search

### PR DESCRIPTION
**Pull request by ramramps**
### Purpose

Task : https://jira.autodesk.com/browse/QNTM-3406

This PR fixes the scroll issue in Incanvas Search.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@alfarok

### CODE REVIEW

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs [hunk 1]
```diff
@@ -1817,7 +1817,11 @@ private void Log(string message)
 
         private void Window_PreviewMouseLeftButtonUp(object sender, MouseButtonEventArgs e)
         {
-            HidePopupWhenWindowDeactivated();
+            //if original sender was scroll bar(i.e Thumb) don't close the popup.
+            if(!(e.OriginalSource is Thumb))
+            {
+                HidePopupWhenWindowDeactivated();
```
Comment (ramramps) [inline src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: https://github.com/DynamoDS/Dynamo/commit/b0700d5e00023e3f883fda5e4869e93ba813ae98
This is the original commit where this code was added. I think, this fixed a bug where the incanvas search was open when you open another view (customnode) or another workspace. 
this check should fix that inappropriate close when mouse is released in the scroll.

Comment (alfarok) [reply src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: Hey @ramramps I just pulled the branch and tested locally.  The in-canvas search looks good and no longer collapses when scrolling.  I am not sure how critical it is but the search window remains open now when switching windows and appears in front of any other content you have open which was I think what you were describing above.  Such as 
image

Comment (ramramps) [reply src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: Thanks @alfarok I will check this

Comment (ramramps) [reply src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: I was able to reproduce this on the master branch. This fix doesn't necessarily change that.

**Pull request merged**